packages/core/src/Utils/Validator/declarative-validation-rules.js

```diff
@@ -126,7 +144,7 @@ const initPreBuildDVRs = () => ({
         func: validGeneral,
         message: localize('Only letters, numbers, space, hyphen, period, and apostrophe are allowed.'),
     },
-    length: { func: validLength, message: localize('You should enter {{value}} characters.', { value: '{{value}}' }) },
+    length: { func: validLength, message: '' },
```

Comment:
Why we don't have any message for this error? please add TODO if it is going to be added later

Reply:
Actually I need to revert this. Nice catch.